packages/ethereum-contracts/contracts/agreements/ConstantFlowAgreementV1.sol

```diff
@@ -44,6 +44,21 @@ contract ConstantFlowAgreementV1 is
      * E_NO_NEGATIVE_ALLOWANCE - sender cannot set a negative allowance
      */
 
+    /**
+     * @dev Default minimum deposit value
+     *
+     * NOTE:
+     * - It may come as a surprise that it is not 0, this is the minimum friction we have in the system for the
+     *   imperfect blockchain system we live in.
+     * - It is related to deposit clipping, and it is always rounded-up when clipping.
+     */
+    uint256 public constant DEFAULT_MINIMUM_DEPOSIT = uint256(uint96(1 << 32));
+    /// @dev Maxim deposit value
```

Comment:
Maximum*

Reply:
Fixed.

And also reverted the deposit maxim value to previous value, it was a mistake.